unit_tests/sources/streams/concurrent/test_concurrent_read_processor.py

```diff
@@ -302,55 +227,6 @@ def test_handle_on_partition_complete_sentinel_yields_status_message_if_the_stre
             )
         ]
         assert messages == expected_messages
-        self._another_stream.cursor.close_partition.assert_called_once()
-
-    @freezegun.freeze_time("2020-01-01T00:00:00")
-    def test_given_exception_on_partition_complete_sentinel_then_yield_error_trace_message_and_stream_is_incomplete(
```

Comment:
This was used in case `close_partition` was failing. We don't call `close_partition` anymore hence this test is useless now